### PR TITLE
Investigate a better way to calculate daily volume

### PR DESCRIPTION
**Pull request by 3mp8r3**

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -233,16 +235,25 @@ export const getEthPrice: () => Promise<number[]> = async () => {
 
   try {
     const oneDayBlock = await getBlockFromTimestamp(utcOneDayBack);
+
+    // const result = await client.query({
+    //   query: ETH_PRICE(),
```
Comment (FriskyHamTitz) [inline src/utils/index.ts:240]: Remove commented out code.

File: src/utils/index.ts [hunk 2]
```diff
@@ -1177,30 +1188,47 @@ export async function getGlobalData(
...
+    //   query: GLOBAL_DATA(oneWeekBlock?.number),
+    //   fetchPolicy: 'network-only',
+    // });
+    // const oneWeekData = oneWeekResult.data.uniswapFactories[0];
+
+    // const twoWeekResult = await client.query({
+    //   query: GLOBAL_DATA(twoWeekBlock?.number),
+    //   fetchPolicy: 'network-only',
+    // });
+    // const twoWeekData = twoWeekResult.data.uniswapFactories[0];
```
Comment (FriskyHamTitz) [inline src/utils/index.ts:1214]: Removed commented out code

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/utils/index.ts (hunk 2 above)
Comment (3mp8r3) [reply src/utils/index.ts]: removed commented code

File: src/utils/index.ts (hunk 1 above)
Comment (3mp8r3) [reply src/utils/index.ts]: removed commented code

File: src/state/stake/hooks.ts [hunk 3]
```diff
@@ -898,7 +900,13 @@ const getOneDayVolume = async () => {
...
       .number,
   );
-  const oneDayOldBlock = current - 45000;
+  const utcCurrentTime = dayjs();
+  const utcOneDayBack = utcCurrentTime
+    .subtract(1, 'day')
+    .startOf('minute')
+    .unix();
+
+  const oneDayOldBlock = await getBlockFromTimestamp(utcOneDayBack);
```
[the remark this answers is not in the document]
Comment (3mp8r3) [reply src/state/stake/hooks.ts:909]: It is using on there. 
image

**Pull request merged**